src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/RoleAssignment.java

```diff
@@ -22,4 +23,18 @@
     private Instant created;
     private List<String> authorisations;
     private RoleAssignmentAttributes attributes;
+
+    /**
+     * This is how the de-serialization works:
+     * Missing caseId -> caseId == null
+     * caseId: null -> Optional.empty()
+     * caseId: "null" -> Optional.empty()
+     * caseId: "12345" -> Optional.of("12345")
+     */
+    public boolean isCaseRoleAssignment() {
+        return this.getAttributes() != null
+            && this.getAttributes().getCaseId() != null
+            && this.getAttributes().getCaseId().isPresent()
+            && StringUtils.isNotBlank(this.getAttributes().getCaseId().get());
```

Comment:
instead of these two lines:             

&& this.getAttributes().getCaseId().isPresent()
            && StringUtils.isNotBlank(this.getAttributes().getCaseId().get());
            
can we use **StringUtils.isNotBlank(this.getAttributes().getCaseId().orElse(""))**